scripts/clusterrequests.py

```
@@ -0,0 +1,585 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import subprocess
+import sys
+from decimal import Decimal, InvalidOperation, ROUND_HALF_UP
+from pathlib import Path
+from typing import Dict, List, Optional, Set, Tuple
```

Ruff (0.15.12)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)